tests/test_google_sheets_pagination.py

```diff
@@ -62,6 +62,7 @@ def test_run(self):
                     # our fake pk value ('id')
                     expected_pk_list = list(range(1, 238))
                     expected_pk_list = [x for x in expected_pk_list if x not in [198, 199]]
+                    expected_pk_list = list(map(str, expected_pk_list))
                     self.assertEqual(expected_pk_list, fake_pk_list)
                     
                     # verify the data for the "sadsheet-pagination" stream is free of any duplicates or breaks by checking
```

Comment:
Because the `fake_pk_list` as well as the `actual_pk_list` both contains string data unlike the `expected_pk_list` which contains integer data.

Reply:
The first time `expected_pk_list` is defined on line 65 it is a list of strings, but the second time it's defined on line 70/71 it is a list of ints. So it looks like `actual_pk_list` has ints while `fake_pk_list` has strings. This inconsistency is concerning because they should be identical columns.

Reply:
You're right, but as we changed from multipleOf to singer.decimal [here](https://github.com/singer-io/tap-google-sheets/pull/74/files#diff-f13d1acaf6325f8a289683dc3247ad13f689af64d585054f7d7be76f237c05b0R166), hence the int would now be fetched as string.

Reply:
Then why are the expectations set to ints? And how is it passing? Either there is a bug in the tap or there is a test issue here.

Reply:
The `fake_pk_list` is the data stored in the sheets(id value) which would now be retrieved as a string due to this change, whereas the `actual_pk_list` is the __sdc_row(actual pk value) which would still be an int value. Hence the expectations would change as per the list

Reply:
Why would an int be typed as a decimal formatted string? Does that mean we are not supporting ints now?

Reply:
We never supported ints, this was python unittest allowing us to say 3.0 == 3. This will not affect the primary key since the schema for _sdc_row is hardcoded. All good 👍